src/main/resources/db/postgres/schema.sql

```diff
@@ -1,102 +1,75 @@
+-- Create tables only if they don't exist
 CREATE TABLE IF NOT EXISTS vets (
-  id SERIAL,
+  id SERIAL PRIMARY KEY,
```

Comment:
**question**: why did you change to this primary key? I suppose the `CONSTRAINT pk_vets PRIMARY KEY (id)` was there in order to provide the `pk_vets` name to the primary key.

Reply:
You are correct. This should be reverted.